benchmarks/pca/solvers/scikit_learn_intelex.py

```diff
@@ -0,0 +1,115 @@
+from importlib.metadata import version
+
+from benchopt import BaseSolver, safe_import_context
+from benchopt.stopping_criterion import SingleRunCriterion
+
+with safe_import_context() as import_ctx:
+    # isort: off
+    import dpctl
+    import dpctl.tensor as dpt
+    import numpy as np
+    from sklearnex.decomposition import PCA
+
+    # isort: on
+
+
+class Solver(BaseSolver):
+    name = "scikit-learn-intelex"
+
+    requirements = [
+        "scikit-learn-intelex",
+        "dpcpp-cpp-rt",
+    ]
+
+    parameters = {
+        "device, runtime": [
+            ("cpu", "numpy"),
+            ("cpu", "level_zero"),
```

Comment:
Does `level_zero` for CPU actually exists?

Reply:
no and apparently it's not meant to, I'll remove this occurence.